src/realm/object-store/sync/impl/sync_metadata.cpp

```diff
@@ -617,6 +622,15 @@ void SyncUserMetadata::set_device_id(const std::string& device_id)
     m_realm->commit_transaction();
 }
 
+SyncUserProfile SyncUserMetadata::profile() const
+{
+    REALM_ASSERT(m_realm);
+    m_realm->verify_thread();
+    m_realm->refresh();
+    StringData result = m_obj.get<String>(m_schema.idx_refresh_token);
```

Comment:
```suggestion
    StringData result = m_obj.get<String>(m_schema.idx_profile_dump);
```
Is this the right `ColKey` field?